### ISSUE TITLE
Kindlegen testing

### ISSUE THREAD

**Issue by unit29868** [unit29868/pressbooks#10]
Test for any Kindlegen issues against new files.

**Comment by bdolor** [comment]
@connerbw — clarification behind the intent on this one. With respect to the current functionality of `function fixMobi()` in assets/js/export.js, are we now wanting to add the epub3 export file as an acceptable, alternative dependency for creating .mobi?

**Comment by dac514** [comment]
Yes. I think so. Ideally EPUB3 replaces EPUB2 - And EPUB2 gets moved under Exotic. 

For this to happen Kindlgen results must work. For it to qualify as "working" this means large swathes of PressBooks (dot com) users, but mostly Hugh, are happy with the outputs. It also means tweaking dozens and dozens of CSS files.We have many private themes not available in the open source version for customers... 

Suffice to say, this is a huge job and I would keep it as the last thing, as it will be an ongoing effort.

**Comment by hughmcguire** [comment]
Agree with Dac. In theory, the new exports should affect the markup, which means in theory the CSS files should all be ok and in theory Kindlegen/MOBI exports should be fine. 

The stress is on the theoretical nature of the theory.

However: EPUB3 capabilities will in theory introduce some new tags we'll have to manage in our CSS ... which will require some testing.

And other things I have no doubt.
